apps/retrospective/serializers/sprint_task.go

```diff
@@ -29,7 +30,7 @@ type SprintTask struct {
 
 // SprintTasksSerializer ...
 type SprintTasksSerializer struct {
-	Tasks []SprintTask
+	Tasks []*SprintTask
```

Comment:
Hey, help me to understand how this changes the behavior?

Reply:
This is done to allow edits in the Task objects in the list if we iterate over them using for loop.